node_modules/col-activities/lib/api.js

```diff
@@ -260,6 +260,13 @@ var getActivitiesForUserId = module.exports.getActivitiesForUserId = function(ct
         return;
       }
 
+      // Omit activities performed by administrators not active in the course.
+      if (activity.actor &&
+          activity.actor.canvas_enrollment_state === 'inactive' &&
+          (_.includes(CollabosphereConstants.ADMIN_ROLES, activity.actor.canvas_course_role))) {
+        return;
+      }
```

Comment:
@paulkerschen questions on conditional above:
1. does it make sense to have the slightly more broad `(state === inactive || is_admin)`?
1. parens around `_includes()` are unnecessary?